site/gatsby-site/src/components/cite/IncidentCard.js

```diff
@@ -67,6 +68,7 @@ const IncidentCard = ({ item, authorsModal, submittersModal, flagReportModal })
             {item.source_domain} &middot;{' '}
             {item.date_published ? item.date_published.substring(0, 4) : 'Needs publish date'}
           </WebArchiveLink>
+          <TranslationBadge className="mx-2" originalLanguage={item.language} />
```

Comment:
I think this looks awkward at the bottom of the card right next to the "show details" button, especially since it's hard to tell that one is supposed to be a button and the other a badge. I think I would put the badge inline at the start of the text, or below or beside the date.

<img src="https://user-images.githubusercontent.com/25443411/183116721-efb105ee-8a2e-471f-a8b0-1753891229bf.png" height="300"/>

Reply:
Yup agree, I moved it to the top, couldn't find an east way to put it inline, probably the show details button should be blue too.